app/models/user.js

```diff
@@ -10,4 +12,23 @@ export default DS.Model.extend({
   token: DS.attr('string'),
   createdAt: DS.attr('string'),
   updatedAt: DS.attr('string'),
+
+  async fetchFeed(page = 1) {
+    const { articles, articlesCount } = await this.session.fetch(`/articles/feed?page=${page}`);
+    if (!articles.length) {
+      return [];
+    }
+    const ids = articles.map(article => article.slug);
+    const normalizedArticles = articles.map(article =>
```

Comment:
The normalization story here could be better but I'm going to punt on making that better for now